R/variables.R

```diff
@@ -338,7 +338,9 @@ create_variables <- function(parameters) {
                    last_boosted_iaa = last_boosted_iaa,
                    iaa = iaa,
                    iam = iam,
-                   hypnozoites = hypnozoites
+                   hypnozoites = hypnozoites,
+                   ls_drug = ls_drug,
```

Comment:
Just a note that in the description (lines 38 and 39) it would be worth stating that these a "liver-stage"